modules/data.land/R/extract_veg.R

```diff
@@ -30,13 +56,19 @@ extract_veg <- function(new_site, start_date, end_date,
  
  # TODO: generalize this as we have more sources that have their own DB like FIA
  if(is.null(format_name)){
+   if(source == "FIA"){
    format_name <- "fia" 
    code_col    <-  "spcd"
+   }else{
+      code_col    <- "species_USDA_symbol"
+      format_name <- "usda"
+      obs[obs$species_USDA_symbol != "2PLANT", ] #removes the rows with 2PLANT, this is a NEON specific code that means they could not identify the species 
+   }
  }
 
  
  # match code to species ID
- spp.info <- match_species_id(input_codes = obs[[code_col]], format_name = format_name)
+ spp.info <- PEcAn.data.land::match_species_id(input_codes = obs[[code_col]], format_name = format_name)
```

Comment:
you shouldn't need to reference functions names by namespace within the package itself.